codegen/module_system.go

```diff
@@ -1116,15 +1117,15 @@ func (g *EndpointGenerator) generateEndpointFile(
 		PackageName: instance.PackageInfo.GeneratedPackagePath + "/workflow",
 		AliasName:   "workflow",
 	})
-	if e.WorkflowImportPath != "" {
+	if e.WorkflowType == "custom" {
```

Comment:
replace the "custom" in this file too :)

Reply:
+1